### PR TITLE
Feature/process

### PR DESCRIPTION
**Pull request by StefanDywersant**
Issue #37

### CODE REVIEW

File: src/client/components/RouteForm/RouteForm.jsx [hunk 1]
```diff
@@ -32,32 +42,50 @@ const DELIVERY_CHANNEL_ID_FORM_MAP = {
...
 };
 
-const TO_SUPPLIER_DOC_TYPES = [ DOC_TYPE.ORDER, DOC_TYPE.RFQ ];
+const TO_SUPPLIER_PROCESSES = [ PROCESS.ORDERING ];
 
-const TO_CUSTOMER_DOC_TYPES = [ DOC_TYPE.INVOICE, DOC_TYPE.MDRTC ];
+const TO_CUSTOMER_PROCESSES = [ PROCESS.INVOICING, PROCESS.MASTER_DATA ];
 
 if (process.env.EXPERIMENTAL_FEATURES_ENABLED === 'true') {
+    PROCESS['INVOICING'] = 'invoicing';
```
Comment (gr4per) [inline src/client/components/RouteForm/RouteForm.jsx]: billing

File: src/client/components/RouteForm/i18n/en.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
 export default {
+    'RouteForm.process.label': 'Process',
+    'RouteForm.process.INVOICING': 'Invoicing',
```
Comment (gr4per) [inline src/client/components/RouteForm/i18n/en.js]: billing

File: src/client/components/RouteForm/i18n/en.js [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
+    'RouteForm.process.ORDERING': 'Ordering',
+    'RouteForm.process.MASTER_DATA': 'Master data',
```
Comment (gr4per) [inline src/client/components/RouteForm/i18n/en.js]: masterdata

and what about fulfillment ? (ReceiptAdvice etc.)

Comment (StefanDywersant) [reply src/client/components/RouteForm/i18n/en.js]: I did not added processes/docTypes for which I don't have any Form prepared. I can do it, but I'd need a list of thing I should support.

**Pull request merged**